ts/protocol/utils.ts

```diff
@@ -132,7 +132,9 @@ export function stringValidator(value: any): boolean {
  * Returns true if the current origin is https.
  */
 export function isSecureOrigin(): boolean {
-  return window.location.protocol === 'https:';
+  return window.location.protocol === 'https:' ||
```

Comment:
Yeah that's fine, you can leave it here for expediency. I missed it in the PR description. Generally I prefer isolating unrelated changes.